### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [franksouza183/Evolvere-Icons#133] [General]
I'd suggest adding a description to the repository~

**Comment by ElectronicsArchiver** [comment]
> An linux icon theme, part of Evolvere Suit (WIP)

You have a small typo, it would be:

`A linux icon theme, part of the Evolvere Suit (WIP)`

Or shorter

`The Evolvere Linux Icon Theme ( WIP )`

**Reply by franksouza183** [reply]
Evolvere Suit is a work in progress, the icon theme too, but on actual status it's "stable" and always will have some new icon to add, technically is also working in progress, but then, every icon theme should also be considered like wip. Evolvere suit just have the icon theme "working"

**Reply by franksouza183** [reply]
Oh, I forgot to fix the typo, done. Thanks!
